pyperformance/cli.py

```diff
@@ -165,15 +167,23 @@ def parse_args():
         sys.exit(1)
 
     if hasattr(options, 'python'):
-        # Replace "~" with the user home directory
-        options.python = os.path.expanduser(options.python)
-        # Try to the absolute path to the binary
-        abs_python = shutil.which(options.python)
-        if not abs_python:
-            print("ERROR: Unable to locate the Python executable: %r" %
-                  options.python)
-            sys.exit(1)
-        options.python = os.path.realpath(abs_python)
+        if options.python is not None:
+            # Replace "~" with the user home directory
+            options.python = os.path.expanduser(options.python)
+            # Try to the absolute path to the binary
+            abs_python = shutil.which(options.python)
+            if not abs_python:
+                print("ERROR: Unable to locate the Python executable: %r" %
+                      options.python)
+                sys.exit(1)
+            options.python = os.path.realpath(abs_python)
+        else:
+            # It would seems like it's possible to put sys.executable as the
+            # default when calling add_argument, but for some reason it returns
+            # another value in some cases: When the python binary is a symbolic
+            # link, the sys.executable in add_argument returns the realpath, while
+            # here it returns the relative path (before following the link).
```

Comment:
I don't understand why this change is needed. sys.executable should not be changed between line 151 and line 186.

Reply:
You're right, the comment I wrote is incorrect. I guess I got confused with the ``if hasattr(options, 'python')`` part.

The issue I have is with the line ``options.python = os.path.realpath(abs_python)`` - this follows the symbolic link from ``venv/bin/python`` to outside the venv.

My modification allows me to run without passing ``--python`` and then it goes into the ``else`` clause which doesn't have the ``realpath`` call.  I can't get around that without modifying the code in some way.

Is there a reason for this ``realpath`` call? Can it be removed? The other lines (replacing "~" and finding the absolute path) are not an issue. I will update the review accordingly.